pages/games/[id].tsx

```diff
@@ -175,45 +189,44 @@ const GameDetailsSidebar = ({
               <Text size="sm">Add this game to your display case!</Text>
             )}
             <Divider my="sm" color="dark.5" />
-            <Select
-              label="Status"
-              placeholder="Pick one"
-              value={status}
-              onChange={(value) => {
-                setStatus(value || "");
-              }}
-              data={[
-                { value: String(GameEntryStatus.WISHLIST), label: "Wishlist" },
-                { value: String(GameEntryStatus.BACKLOG), label: "Backlog" },
-                { value: String(GameEntryStatus.PLAYING), label: "Playing" },
-                {
-                  value: String(GameEntryStatus.COMPLETED),
-                  label: "Completed",
-                },
-                { value: String(GameEntryStatus.DROPPED), label: "Dropped" },
-              ]}
-            />
-            <Select
-              label="Rating"
-              placeholder="Out of 10"
-              value={rating}
-              onChange={(value) => {
-                setRating(value || "0");
-              }}
-              data={Array(11)
-                .fill(0)
-                .map((_, index) => `${index}`)}
-            />
-            <Select
-              label="Platform"
-              placeholder="Pick one"
-              value={platform}
-              onChange={(value) => {
-                setPlatform(value || "");
-              }}
-              data={game?.platforms || []}
-            />
-            <Button onClick={submitGameEntry}>Update</Button>
+            <form onSubmit={form.onSubmit(submitGameEntry)}>
+              <Select
+                label="Status"
+                placeholder="Pick one"
+                data={[
+                  {
```

Comment:
I think it's possible to map `GameEntryStatus` enum into this type of `data`. But we just leave it like this for now, since I don't think we'll be changing the game status anyway

Reply:
Cuz we're storing as ints for transfer with backend. So we gonna need this somewhere at least